packages/gasket-plugin-elastic-apm/lib/actions.js

```diff
@@ -0,0 +1,21 @@
+/** @type {import('@gasket/core').HookHandler<'actions'>} */
+module.exports = function actions(gasket) {
+  return {
+    async getApmTransaction(req) {
+      const apm = gasket.apm;
```

Comment:
I'm not seeing how `gasket.apm` is ever assigned.

Reply:
You still have other places looking for `gasket.apm`, as well as some tests setting it.